backend/corpora/dataset_processing/remaster_cxg.py

```diff
@@ -27,48 +32,151 @@ def process(dataset_id: str, cellxgene_bucket: str, prefix=None, dry_run=True):
         dataset_path = get_bucket_prefix(dataset_id)
         object_key = f"{dataset_path}.cxg"
     path = f"s3://{cellxgene_bucket}/{object_key}/X"
+    obs_path = f"s3://{cellxgene_bucket}/{object_key}/obs"
+    meta_path = f"s3://{cellxgene_bucket}/{object_key}/cxg_group_metadata"
 
     logger.info(f"Processing dataset at path {path}, dry run {dry_run}")
 
-    local_path = "/cxg"
     logger.info(f"Downloading {path} to {local_path}/X_old")
-
-    download_command = ["aws", "s3", "sync", path, f"{local_path}/X_old"]
+    download_command = ["aws", "s3", "sync", path, f"{local_path}/X_old", "--quiet"]
     # Let errors fail the pipeline
     subprocess.run(download_command, check=True)
 
-    params = {
-        "kind": "auto",
-        "obs_extent": 256,
-        "var_extent": 2048,
-        "cell_order": "row",
-        "tile_order": "col",
-        "capacity": 128000,
-        "compression": 22,
-        "target_array": "X_new",
-        "source_array": "X_old",
-        "sparse_threshold": 25.0,
-    }
-
-    compute(cxg=local_path, **params)
-
-    logger.info(f"Dataset at {path} computed successfully")
+    logger.info(f"Downloading {obs_path} to {local_path}/old_obs")
+    download_command = ["aws", "s3", "sync", obs_path, f"{local_path}/old_obs", "--quiet"]
+    # Let errors fail the pipeline
+    subprocess.run(download_command, check=True)
 
-    if not dry_run:
-        upload_command = ["aws", "s3", "sync", "--delete", f"{local_path}/X_new", path]
-        subprocess.run(upload_command, check=True)
+    logger.info(f"Downloading {meta_path} to {local_path}/cxg_group_metadata")
+    download_command = ["aws", "s3", "sync", meta_path, f"{local_path}/cxg_group_metadata", "--quiet"]
+    # Let errors fail the pipeline
```

Comment:
currently, on errors, we route all failures to a HandleErrors lambda which will update the processing status to FAILED. In this case, we don't want to do that. Let's wrap a 'try-catch' around remaster_cxg.process in the main function in process.py, in which we just log the error w/o re-raising (or just `pass` if logging isn't needed for the use case) so that errors don't get routed to HandleErrors